pages/index.html

```diff
@@ -4,208 +4,37 @@
-  with support from nine sponsor organizations.
-  Over the next year,
-  he recorded 120 short video lessons and ran half a dozen week-long classes for his backers.
-</p>
-<p>
-  This version of Software Carpentry was much more successful than its predecessors,
-  in part because the scientific landscape itself had changed.
-  Open access publishing, citizen science, and dozens of other innovations
-  had convinced scientists that they needed to be able to do more than just crunch numbers.
-  We also made contact with like-minded organizations,
-  particularly <a href="https://thehackerwithin.github.io/">The Hacker Within</a>,
-  who showed us that intensive two-day workshops worked better than week-long classes.
-  Putting this all together led to our current model:
-</p>
-<ul>
-  <li>
-    <p>
-      Curriculum is developed and improved in a public repository
-      using methods borrowed from the open source software community.
-    </p>
-  </li>
-  <li>
-    <p>
-      Instructors volunteer their time,
-      while workshop host sites cover their travel and accommodation costs.
-    </p>
-  </li>
-  <li>
-    <p>
-      Instructors use live coding instead of slides
-      while learners following along on their own machines.
-    </p>
-  </li>
-</ul>
-<h2>Scaling Up</h2>
-<p>
-  In the fall of 2011,
-  Wilson worked with the Mozilla Foundation to prepare a grant to the Sloan Foundation
-  to put Software Carpentry on a more stable financial footing.
-  That grant,
-  awarded in January 2012,
-  and a second one later that year
-  paid for Wilson and some administrative support,
-  which allowed us to increase the number of workshops.
-  In turn,
-  that growth led to us starting a training program
-  to teach instructors the basics of educational psychology and instructional design.
-</p>
-<h2>The Present Day</h2>
-<p>
-  Wilson left Mozilla in July 2014 to help found the Software Carpentry Foundation,
-  an independent non-profit volunteer sponsored by <a href="http://numfocus.org">NumFOCUS</a>.
-  Software Carpentry's governing body is a <a href="{{site.baseurl}}/scf/">Steering Committee</a>,
-  which is elected from and by its <a href="{{site.baseurl}}/scf/members/">members</a>
-  and assisted by an Advisory Board made up of representatives from partner organizations.
-  The Foundation's <a href="{{site.baseurl}}/scf/#committee-2015">first Steering Committee</a>
-  was elected in January 2015,
-  and in October 2015
-  <a href="{{site.baseurl}}/team/#duckles.jonah">Jonah Duckles</a> began work
-  as the Foundation's new Executive Director.
+  Having started in 1998,
+  Software Carpentry is now a lesson program within <a href="{{site.carpentries_url}}">The Carpentries</a>.
+  Its focus is on the the computing skills researchers need to get more done in less time and with less pain,
+  and its <a href="{{site.carpentries_url}}/instructors/">volunteer instructors</a>
```

Comment:
```suggestion
  and its <a href="{{site.carpentries_url}}/instructors/">volunteer Instructors</a>
```

I think we should use an initial cap here